README.md

```diff
@@ -108,12 +108,33 @@ smb2Client.mkdir('path\\to\\the\\directory', function(err) {
 });
 ```
 
-> `smb2Client.readdir ( path, callback )`
+> `smb2Client.readdir ( path, [options], callback )`
+
+- `path` String
+- `options` Object
+  - `encoding` String | Null default = null
+- `callback` Function
 
 Asynchronous `readdir(3)`: reads the contents of a directory.
 
 The result is an array of the names of the files in the directory excluding `'.'` and `'..'`.
 
+If you want the response to include stats, you need to pass the `stats: true`. Response will be an Array of this form:
+
+```
+[
+    {
+        Filename: String,
```

Comment:
`Filename` or `filename`?

Also, why not using name as Node does in [`fs.Dirent`](https://nodejs.org/dist/latest/docs/api/fs.html#fs_class_fs_dirent)?

Reply:
ok, that was a typo indeed. 

But fair enough, I will change it to name, which makes more sense with entries that are directories